ext/standard/array.c

```diff
@@ -1029,11 +1029,54 @@ static inline HashTable *get_ht_for_iap(zval *zv, bool separate) {
 	return zobj->handlers->get_properties(zobj);
 }
 
+enum ia_undef_strategy {
```

Comment:
```suggestion
enum php_array_iter_undef_strategy {
```
I kind of prefer prefix php things with `php` so it's clear where it comes from and prevents theoretical naming conflicts in the future.